javatests/arcs/core/data/CapabilitiesTest.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.data
+
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
```

Comment:
Prefer using `Truth`'s `assertThat(condition).is[ExpectedCondition]()` system of test assertions over JUnit's.

Reply:
done